core/services/aws/tests/user/updateUser.test.ts

```diff
@@ -49,8 +49,8 @@ describe('updateUserLambda', () => {
       lastVaccinatedDate: '2023-05-01'
     }
 
-    const mockResponse = 'User updated successfully'
-    mockedUserService.prototype.updateUser.mockResolvedValue(mockResponse)
+    const mockResponse = 'Your Profile info has been updated.'
```

Comment:
shouldn't it be imported from the consts?

Reply:
Updated in f98450b71743d5473aa397d4854e7347002cc21c